packages/inter-protocol/src/auction/scheduler.js

```diff
@@ -116,42 +117,103 @@ export const computeRoundTiming = (params, baseTime) => {
  * @property {() => void} startRound
  */
 
+/**
+ * @typedef {object} ScheduleNotification
+ *
+ * @property {Timestamp | undefined} activeStartTime start time of current
+ *    auction if auction is active
+ * @property {Timestamp} nextStartTime start time of next auction
+ * @property {Timestamp} nextDescendingStepTime when the next descending step
+ *    will take place
+ */
+
 /**
  * @param {AuctionDriver} auctionDriver
  * @param {import('@agoric/time/src/types').TimerService} timer
  * @param {Awaited<import('./params.js').AuctionParamManaager>} params
  * @param {import('@agoric/time/src/types').TimerBrand} timerBrand
+ * @param {Publisher<ScheduleNotification>} schedulePublisher
  */
 export const makeScheduler = async (
   auctionDriver,
   timer,
   params,
   timerBrand,
+  schedulePublisher,
 ) => {
   /**
    * live version is defined when an auction is active.
    *
    * @type {Schedule | undefined}
    */
   let liveSchedule;
-  /**
-   * Next should always be defined after initialization unless it's paused
-   *
-   * @type {Schedule | undefined}
-   */
-  let nextSchedule;
+
+  /** @returns {Promise<{now:Timestamp, nextSchedule: Schedule}>} */
```

Comment:
nit: space around object members
```suggestion
  /** @returns {Promise<{ now: Timestamp, nextSchedule: Schedule }>} */
```

Reply:
done